cnd/src/main.rs

```diff
@@ -48,8 +50,10 @@ fn main() -> anyhow::Result<()> {
         process::exit(0);
     }
 
-    let base_log_level = settings.logging.level;
-    logging::initialize(base_log_level, settings.logging.structured)?;
+    // We still want upstream library log messages, just only at Info level.
+    LogTracer::init_with_filter(LevelFilter::Info)?;
```

Comment:
I would see this as part of the initialization, no?
Why add this level of detail to `main`?

Reply:
Good point, thanks.  Implemented.